add-on/preferences/prefs.js

```diff
@@ -33,11 +33,7 @@ pref("loop.CSP", "default-src 'self' about: file: chrome:; img-src * data:; font
 pref("loop.fxa_oauth.tokendata", "");
 pref("loop.fxa_oauth.profile", "");
 pref("loop.support_url", "https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/cobrowsing");
-#ifdef LOOP_BETA
 pref("loop.facebook.enabled", true);
```

Comment:
oh, I see it now on the invitation view. 
Still, we're checking a pref that is always `true`, which is weird, plus we'll need another bug eventually to get rid of the rests of it. We could remove it all, and back it up the whole thing if needed, instead of splitting it.

Anyway, if we want to go conservative, I'm happy with it as long as we don't forget about the remaining code (followup?)